common/cmake/devices.cmake

```diff
@@ -1491,7 +1545,12 @@ function(ADD_FPGA_TARGET)
 
   get_target_property_required(VPR env VPR)
 
-  get_target_property_required(ROUTE_CHAN_WIDTH ${ARCH} ROUTE_CHAN_WIDTH)
+  # Use route channel width from the device. If not provided then use the
+  # one from the arch.
+  get_target_property(ROUTE_CHAN_WIDTH ${DEVICE} ROUTE_CHAN_WIDTH)
```

Comment:
Is this change still required?

Reply:
Currently yes. There is an ongoing effort to use externally supplied rr graphs. With that done specifying channel width per device will no longer be necessary.